ThunderCloud/UIApplication+Extensions.swift

```diff
@@ -13,7 +13,10 @@ extension UIApplication {
     
     /// First `UIWindow` where `isKeyWindow`
     var appKeyWindow: UIWindow? {
-        return windows.first { $0.isKeyWindow }
+        return UIApplication.shared.connectedScenes
```

Comment:
Does is app guaranteed to have scenes?

Reply:
The older apps don't have scenes, will revert this.